rest/model/task_annotations.go

```diff
@@ -155,3 +161,28 @@ func ArrAPIIssueLinkArrtaskannotationsIssueLink(t []APIIssueLink) []annotations.
 	}
 	return m
 }
+
+func GetJiraTickets(issuelinks []APIIssueLink) ([]APIIssueLink, error) {
+	settings := evergreen.GetEnvironment().Settings()
+	jiraHandler := thirdparty.NewJiraHandler(*settings.Jira.Export())
+	catcher := grip.NewBasicCatcher()
+
+	var res []APIIssueLink
+	for _, issue := range issuelinks {
+		urlObject, err := url.Parse(*issue.URL)
+		if err != nil {
+			catcher.Add(errors.Wrap(err, "problem parsing issue string"))
```

Comment:
Can use catcher.Wrap. Also, it's the URL specifically, not like the issue name, can this error message be clarified?